reports/Connecticut.md

```diff
@@ -0,0 +1,21 @@
+## Waterbury
+
+
+### Cop | June 3rd
```

Comment:
The name of this incident is a bit short and could clash with other incident names, it might be better to include a slightly longer name like
```suggestion
### Officer forces women into car and wrestles with another | June 3rd
```